airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_full_refresh.py

```diff
@@ -67,9 +67,8 @@ def test_sequential_reads(
                 serializer = partial(make_hashable, exclude_fields=ignored_fields.get(stream))
             stream_records_1 = records_by_stream_1.get(stream)
             stream_records_2 = records_by_stream_2.get(stream)
-            # Using
-            output_diff = set(map(serializer, stream_records_1)).symmetric_difference(set(map(serializer, stream_records_2)))
-            if output_diff:
+            if not set(map(serializer, stream_records_1)).issubset(set(map(serializer, stream_records_2))):
+                output_diff = set(map(serializer, stream_records_1)).symmetric_difference(set(map(serializer, stream_records_2)))
```

Comment:
do we still need to check the symmetric difference if we're effectively doing the validation above by verifying that `records_1` is not a subset of `records_2`?

Thinking about all the permutations, `records_2` should at minimum have the same records as `records_1` but could have more. So I think your above condition is sufficient `set(map(serializer, stream_records_1)).issubset(set(map(serializer, stream_records_2)))`

If we've gotten into this block where we know that `records_1` has records that `records_2` is missing, maybe we should be more descriptive and use `records_1 - records_2` to show which records `records_2` was missing that should have been there. The symmetric difference combines it all together which might be confusing.